xen/drivers/passthrough/arm/scmi_smc_maker.c

```diff
@@ -0,0 +1,350 @@
+/*
+ * scmi_process.c
```

Comment:
Whole file comments:
Licence/copyright.
re-check.
Could you name files to scmi_dt_maker.c?
By the way It doesn't matter what is Xen2ATF interface - the virtual interfaces can be only SMC/HVC based. 
could name all function/items in this file as scmi_dt_c
check statics

Reply:
fixed